docs/Guides/Ecosystem.md

```diff
@@ -608,6 +608,7 @@ section.
   and lightweight Sequelize plugin for Fastify.
 - [`typeorm-fastify-plugin`](https://github.com/jclemens24/fastify-typeorm) A simple
   and updated Typeorm plugin for use with Fastify.
+- [`fastify-list-routes`](https://github.com/chuongtrh/fastify-list-routes) A simple plugin for Fastify list all available routes.
```

Comment:
Can you move into `Community Plugin`? Core plugin means the plugins maintained inside the `Organization`.